File: gui/src/main/java/com/woohyman/gui/ui/multitouchbutton/MultitouchBtnInterface.java
package com.woohyman.gui.ui.multitouchbutton;

import android.view.MotionEvent;

import nostalgia.framework.ui.multitouchbutton.OnMultitouchEventListener;

public interface MultitouchBtnInterface {
    void onTouchEnter(MotionEvent event);

    void onTouchExit(MotionEvent event);

    void setOnMultitouchEventlistener(OnMultitouchEventListener listener);

    int getId();

    boolean isPressed();

    void requestRepaint();

    void removeRequestRepaint();

    boolean isRepaintState();

    int getVisibility();

}

File: gui/src/main/java/com/woohyman/gui/ui/multitouchbutton/MultitouchBtnInterface.kt
package com.woohyman.gui.ui.multitouchbutton

import android.view.MotionEvent

interface MultitouchBtnInterface {
    fun onTouchEnter(event: MotionEvent?)
    fun onTouchExit(event: MotionEvent?)
    fun setOnMultitouchEventlistener(listener: OnMultitouchEventListener?)
    fun getId(): Int
    fun isPressed(): Boolean
    fun requestRepaint()
    fun removeRequestRepaint()
    val isRepaintState: Boolean
    fun getVisibility(): Int
}